assets/js/components/DataBlockGroup.js

```diff
@@ -96,11 +98,30 @@ export default function DataBlockGroup( { className, children } ) {
 		} );
 	};
 
+	const resetFontSizes = async ( blocks ) => {
```

Comment:
@ankitrox @benbowler, it's not clear why this needs to be `async`, although I assume it's something to do with the debounce below. Please can you explain why, and add a comment to the code to make it clear?